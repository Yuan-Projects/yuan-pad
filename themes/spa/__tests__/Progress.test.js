import renderer from 'react-test-renderer';
import Progress from '../src/progress';

it('Progress bar renders correctly', () => {
  const tree = renderer.create(<Progress loadingModalIsOpen={true} />).toJSON();
  expect(tree).toMatchSnapshot();

  const tree2 = renderer.create(<Progress loadingModalIsOpen={false} />).toJSON();
  expect(tree2).toMatchSnapshot();
});