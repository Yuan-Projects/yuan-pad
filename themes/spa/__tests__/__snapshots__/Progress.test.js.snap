// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress bar renders correctly 1`] = `null`;

exports[`Progress bar renders correctly 2`] = `null`;
